dali/kernels/signal/fft/stft_gpu_impl.cuh

```diff
@@ -37,7 +37,7 @@ namespace fft {
 class StftImplGPU {
  public:
   StftImplGPU() = default;
-  StftImplGPU(StftImplGPU &&) = default;
+  StftImplGPU(StftImplGPU &&) = delete;
```

Comment:
Why?

Reply:
Cause it's implicitly deleted explicitly defaulted constructor.